app/(tabs)/ProfileView/FavoriteSpotsView.tsx

```diff
@@ -11,9 +12,10 @@ interface FavouriteSpot {
 const favouriteSpots: FavouriteSpot[] = [
```

Comment:
Same with this one
const { favouriteRestaurants } = useContext(App context);
For the real data (once the API calls are implemented)